tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -17,7 +17,7 @@ LOG_MODULE_REGISTER(main);
 #include <drivers/spi.h>
 
 #define SPI_DRV_NAME	CONFIG_SPI_LOOPBACK_DRV_NAME
-#define SPI_SLAVE	CONFIG_SPI_LOOPBACK_SLAVE_NUMBER
+#define SPI_SLAVE_NUMBER	CONFIG_SPI_LOOPBACK_SLAVE_NUMBER
 #define SLOW_FREQ	CONFIG_SPI_LOOPBACK_SLOW_FREQ
 #define FAST_FREQ	CONFIG_SPI_LOOPBACK_FAST_FREQ
```

Comment:
Please keep vertikal alignment for CONFIG_*

Reply:
sure! done